lists/Washington_DC_2024.list

```diff
@@ -0,0 +1,17 @@
+# Dusk
+Welcome
```

Comment:
Noting that I didn't reuse an existing welcome speech, but there wasn't anything particularly novel in the one I ended up doing (just cribbed a bunch of stuff together from existing ones), so I wasn't sure how to notate this